src/db.ts

```diff
@@ -242,38 +224,30 @@ export class Database {
    *   pending PR. It rejects to if the database query fails.
    */
   async getPendingPR(repo: Repo, upstreamRepo: Repo, branch: string): Promise<PendingPR | null> {
-    return new Promise((resolve, reject) => {
-      const sql = `
-                SELECT upstream_pr_ids, action, pr_id, github_issue
-                FROM pending_prs
-                WHERE repo_id=? AND upstream_repo_id=? AND branch=?
-            `;
-      this.get(sql, [repo.id, upstreamRepo.id, branch])
-        .then((result) => {
-          if (!result) {
-            resolve(null);
-            return;
-          }
+    const sql = `
+      SELECT upstream_pr_ids, action, pr_id, github_issue
+      FROM pending_prs
+      WHERE repo_id=? AND upstream_repo_id=? AND branch=?
+    `;
+    const result = await this.get(sql, [repo.id, upstreamRepo.id, branch]);
+    if (!result) {
+      return null;
+    }
 
-          const upstreamPRIDs = [];
-          for (const idStr of (result.upstream_pr_ids as string).split(",")) {
-            upstreamPRIDs.push(parseInt(idStr));
-          }
+    const upstreamPRIDs = [];
+    for (const idStr of (result.upstream_pr_ids as string).split(",")) {
+      upstreamPRIDs.push(parseInt(idStr));
+    }
```

Comment:
Nit: I think this could also work, and is more "functional", which is very hip nowadays.
```suggestion
    const upstreamPRIDs = (result.upstream_pr_ids as string).split(",").map((idStr) => parseInt(idStr))
```